### PR TITLE
CoC update blog post

### PR DESCRIPTION
**Pull request by yabellini**
- Do we also need to mention the changes on the CoC Committee?

### CODE REVIEW

**Comment by mpadge** [comment]
> * Do we also need to mention the changes on the CoC Committee?

I think so, yes. I think the title could stay the same, because the CoC itself is the bit that most affects the blog audience, but the post could include an additional section announcing updated committee membership. @karawoo What do you think?

File: content/blog/2022-07-11-coc-update/index.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+- Greater detail about acceptable and unacceptable behaviors in online settings. 
+- First translation of the text to Spanish.
+
+You can access now the last version full-text in [English](/code-of-conduct/) and [Spanish](/codigo-de-conducta/). 
+
+We welcome your feedback by email to `conduct at ropensci.org`. 
+
+As always our Code of Conduct uses community sources to improve.  For these changes, we have consulted and adapted the [useR! 2021 Code of Conduct](https://user2021.r-project.org/participation/coc/). 
+
+
```
Comment (yabellini) [inline content/blog/2022-07-11-coc-update/index.md:29]: ### Committee Changes

We thanks [Megan Carter](https://www.esipfed.org/about/people/megan-carter) for serving as independent community member until June, 2022 and welcome back [Kara Woo](https://karawoo.com/) to serve on this role.  [Yanina Bellini Saibene](/author/yanina-bellini-saibene/) join the committee as the new rOpenSci Community Manager.

**Review by yabellini — COMMENTED** [review]
I added the changes on the committee

**Comment by mpadge** [comment]
@yabellini You'll also now need to rename the directory (from `07-11` to `07-12` (or whatever). Then you should feel free to merge and publish. Right @maelle?

**Pull request merged**